src/JPEGView/PNGWrapper.cpp

```diff
@@ -36,10 +41,8 @@
  * 3. This notice may not be removed or altered from any source distribution.
  *
  */
-#include <stdlib.h>
-#include "png.h"
 
-struct FrameData {
+struct PngReader::png_cache {
```

Comment:
Thanks for cleaning up the caches implementations for the older caches in the code!  I was recently jumping through the code for the XP support and realized that, while I commented on JXL implementation at the time, the rest of the code, even the legacy webp stuff used the file-based scoping 🙄.